app/communication/models/banner.py

```diff
@@ -0,0 +1,34 @@
+import uuid
+
+from django.db import models
+from django.utils import timezone
+
+from app.common.enums import AdminGroup
+from app.common.permissions import BasePermissionModel
+from app.util.models import BaseModel, OptionalImage
+from app.util.utils import now
+
+
+class Banner(BaseModel, OptionalImage, BasePermissionModel):
+    write_access = AdminGroup.admin()
+    read_access = AdminGroup.admin()
```

Comment:
Her må du nok tenke litt annerledes. Tanken med bannere er at vi skal kunne lagge til informasjon på forsiden dynamisk når vi ønsker det, for eks der jeg har markert på bildet. Altså må **alle** kunne hente ut bannere som er `is_visible==True`

<img width="1120" alt="image" src="https://user-images.githubusercontent.com/31009729/152132741-8dbe4b17-ab71-4bb4-9a1d-cad626c01eea.png">